src/lib/address/unstoppable-domains.ts

```diff
@@ -0,0 +1,71 @@
+import { btcChainId, ethChainId } from '@shapeshiftoss/caip'
+import { Resolution } from '@unstoppabledomains/resolution'
+import { getConfig } from 'config'
+import last from 'lodash/last'
+import {
+  ResolveVanityAddress,
+  ReverseLookupVanityAddress,
+  ValidateVanityAddress,
+} from 'lib/address/address'
+import { logger } from 'lib/logger'
+
+const moduleLogger = logger.child({ namespace: ['unstoppable-domains'] })
```

Comment:
Do we want to be more accurate in the structured logging here e.g `Lib` `Address` `unstoppable-domains` ?